src/Integration.UnitTests/LocalServices/ProjectSystemHelperTests.cs

```diff
@@ -167,6 +168,61 @@ public void ProjectSystemHelper_IsFileInProject_DifferentCase()
             this.testSubject.IsFileInProject(project, newFile).Should().BeTrue();
         }
 
+        [TestMethod]
+        public void ProjectSystemHelper_FindFileInProject_Missing_ReturnsNull()
+        {
+            var project = this.solutionMock.AddOrGetProject("project");
+            project.AddOrGetFile("fileThatExists.txt");
+
+            this.testSubject.FindFileInProject(project, "fileNotInTheProject.txt").Should().BeNull();
+        }
+
+        [TestMethod]
+        public void ProjectSystemHelper_FindFileInProject_ArgChecks()
+        {
+            // 1. Invalid project
+            Action act = () => testSubject.FindFileInProject(null, "file");
+            act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("project");
+
+            // 2. Null file name
+            act = () => this.testSubject.FindFileInProject(new ProjectMock("project"), null);
+            act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("file");
+
+            // 3. Empty file name
+            act = () => this.testSubject.FindFileInProject(new ProjectMock("project"), "");
+            act.Should().ThrowExactly<ArgumentNullException>().And.ParamName.Should().Be("file");
+
+            // 4. Null file name
```

Comment:
The comment is misleading, maybe "whitespace file name"?

Reply:
I copied and pasted this from the `AddFileToProject` test. I'll fix both.